conda_smithy/azure_ci_utils.py

```diff
@@ -0,0 +1,191 @@
+import os
+import typing
+
+from vsts.vss_connection import VssConnection
+from msrest.authentication import BasicAuthentication
+
+
+AZURE_TEAM_INSTANCE = os.getenv("AZURE_INSTANCE", "https://dev.azure.com/conda-forge")
+AZURE_PROJECT_ID = os.getenv("AZURE_PROJECT_ID", "feedstock-builds")
+AZURE_SERVICE_ENDPOINT_NAME = os.getenv("AZURE_SERVICE_ENDPOINT", "conda-forge")
+
+try:
+    # Create a token at https://circleci.com/account/api. Put it in circle.token
```

Comment:
The comment seemed unnecessary since the error message already explains this (and does a better job). So just added PR ( https://github.com/conda-forge/conda-smithy/pull/923 ) to drop this comment (and another one just like it).